types/mongodb/index.d.ts

```diff
@@ -240,57 +241,120 @@ export interface MongoClientOptions extends
     SocketOptions,
     SSLOptions,
     HighAvailabilityOptions {
-    // The logging level (error/warn/info/debug)
+
+    /**
+     * The logging level (error/warn/info/debug)
+     */
     loggerLevel?: string;
-    // Custom logger object
-    logger?: Object;
-    // Default: false;
+
+    /**
+     * Custom logger object
+     */
+    logger?: Object | log;
+
+    /**
+     * Validate MongoClient passed in options for correctness.
+     * Default: false
+     */
     validateOptions?: Object;
-    // The name of the application that created this MongoClient instance.
-    appname?: string;
+
+    /**
+     * The name of the application that created this MongoClient instance.
+     */
+     appname?: string;
+
+     /**
+      * Authentifikation credentials
+      */
     auth?: {
+        /**
+         * The username for auth
+         */
         user: string;
+        /**
+         * The password for auth
+         */
         password: string;
-    },
-    // Determines whether or not to use the new url parser
-    useNewUrlParser?: boolean
+    };
+
+    /**
```

Comment:
Is it necessary to pollute this change with what are effectively stylistic changes?

Reply:
There was bunch of various styles how identifiers are documented, one were documented with line comments others used block comments and my intention was it to unifies these comments.

I decide to use doc comments for the following reason: To my knowledge the most ideas only supports docs comments (block comments) for rendering in editor documentation. For example vscode don't displays the property/method documentation, if it was documented with one line comments.

Reply:
If you mean the replacement of the comma with the semicolon, as i said it's was my intent to unify the code. All other ObjectType definition uses semicolons.

Reply:
I think the typing benefits from the unification of documentation styling.

Reply:
In the future it would be better to do the stylistic changes as separate commits (even if in the same PR). Makes it easier to review!